File: case_studies/argo/read.R
library(readr)

dpath = "/home/kshedden/data/Teaching/argo/julia"

Sys.setenv(VROOM_CONNECTION_SIZE=1000000)

lat = read_csv(file.path(dpath, "lat.csv.gz"))
lat = as.vector(lat[,1])$Column1

lon = read_csv(file.path(dpath, "lon.csv.gz"))
lon = as.vector(lon[,1])$Column1

date = read_csv(file.path(dpath, "date.csv.gz"))
date = as.vector(date[,1])$Column1
day = date - min(date)

pressure = read_csv(file.path(dpath, "pressure.csv.gz"))
pressure = as.vector(pressure[,1])$Column1

temp = read_csv(file.path(dpath, "temp.csv.gz"))
temp = as.matrix(temp)

#psal = read_csv(file.path(dpath, "psal.csv.gz"))
#psal = as.matrix(psal)

File: case_studies/argo/read.py
import pandas as pd
import numpy as np
import os

dpath = "/home/kshedden/data/Teaching/argo/python"

lat = np.loadtxt(os.path.join(dpath, "lat.csv.gz"))
lon = np.loadtxt(os.path.join(dpath, "lon.csv.gz"))

date = np.loadtxt(os.path.join(dpath, "date.csv.gz"), dtype="str")
date = pd.to_datetime(date)
day = date - date.min()
day = np.asarray([x.days for x in day])

temp = np.loadtxt(os.path.join(dpath, "temp.csv.gz"))
pressure = np.loadtxt(os.path.join(dpath, "pressure.csv.gz"))
psal = np.loadtxt(os.path.join(dpath, "psal.csv.gz"))
